mozilla/blurts-server
# MNTOR-2833

# References:

Jira: MNTOR-2833

# Description
Duplicated text “Subscribe to ⁨Monitor Plus⁩ and we’ll and we’ll scan..."
<img width="1699" alt="Screenshot 2024-07-05 at 11 25 43 AM" src="https://github.com/mozilla/blurts-server/assets/31937894/412f5426-367c-4256-a5bc-fd749a4cd654">

# How to test
1. In Storybook navigate to Pages/Logged in/Dashboard/US user, without Premium, with 0 scan results, with all breaches resolved
2. Confirm text has been updated (as seen in this screenshot)
<img width="1705" alt="Screenshot 2024-07-05 at 11 26 12 AM" src="https://github.com/mozilla/blurts-server/assets/31937894/e45fcdf4-97bf-4d99-8cec-d912e2971dd0">

# Checklist (Definition of Done)
